bin/hook-docs/data/actions.json

```diff
@@ -81,9 +129,97 @@
                 "long_description_html": ""
             }
         },
+        {
+            "name": "woocommerce_archive_description",
+            "file": "BlockTypes/LegacyTemplate.php",
+            "type": "action",
+            "doc": {
+                "description": "Hook: woocommerce_archive_description.",
+                "long_description": "",
+                "tags": [
+                    {
+                        "name": "hooked",
+                        "content": "woocommerce_taxonomy_archive_description - 10"
+                    },
+                    {
+                        "name": "hooked",
+                        "content": "woocommerce_product_archive_description - 10"
+                    }
+                ],
+                "long_description_html": ""
+            }
+        },
+        {
+            "name": "woocommerce_before_main_content",
+            "file": "BlockTypes/LegacyTemplate.php",
+            "type": "action",
+            "doc": {
+                "description": "Woocommerce_before_main_content hook.",
+                "long_description": "",
+                "tags": [
+                    {
+                        "name": "hooked",
+                        "content": "woocommerce_output_content_wrapper - 10 (outputs opening divs for the content)"
+                    },
+                    {
+                        "name": "hooked",
+                        "content": "woocommerce_breadcrumb - 20"
+                    }
+                ],
+                "long_description_html": ""
+            }
+        },
+        {
+            "name": "woocommerce_before_main_content",
```

Comment:
Again this seems to be the same as the one above, with a minor difference in the description and an extra tag. Just checking this is intended?